firefox_puppeteer/firefox_puppeteer/api/appinfo.py

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from firefox_puppeteer.base import BaseLib
+from ..base import BaseLib
```

Comment:
Be sure to test this in all environments. We had trouble with relative imports in media_tests when we first got off the ground.